robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowViewTreeObserver.java

```diff
@@ -14,6 +14,7 @@
 @Implements(ViewTreeObserver.class)
 public class ShadowViewTreeObserver {
```

Comment:
Just delete the file I think, and the test :-)
On Feb 23, 2016 12:07 PM, "choesther" <notifications@github.com> wrote:

> In
> robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowViewTreeObserver.java
> <https://github.com/robolectric/robolectric/pull/2306#discussion_r53841483>
> :
>
> > @@ -14,6 +14,7 @@
> >  @Implements(ViewTreeObserver.class)
> >  public class ShadowViewTreeObserver {
>
> I'm in agreement with that. What is the approach for removing shadows?
> … <#-1230887866_>
> On Tue, Feb 23, 2016 at 9:20 AM Jonathan Gerrish <notifications@github.com>
> wrote: In
> robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowViewTreeObserver.java
> <#2306 (comment)
> <https://github.com/robolectric/robolectric/pull/2306#discussion_r53815091>>
> : > @@ -14,6 +14,7 @@ > @Implements <https://github.com/Implements>(ViewTreeObserver.class)
> > public class ShadowViewTreeObserver { I think I would prefer to remove
> this whole shadow rather than reimplementing Android behaviour. You can
> fire the listeners yourselfusing dispatchOnPreDraw() and
> dispatchOnGlobalLayout(). I'm not sure if there is really a need to get
> hold of the listeners themselves, but if there is we could just make a
> non-android accessor that gets the field by reflection. Thoughts? — Reply
> to this email directly or view it on GitHub <
> https://github.com/robolectric/robolectric/pull/2306/files#r53815091>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robolectric/robolectric/pull/2306/files#r53841483>.
>